tycho-extras/pom.xml

```diff
@@ -28,7 +28,7 @@
 	<description>Additional tools for Tycho</description>
 
 	<properties>
-		<jgit-version>5.12.0.202106070339-r</jgit-version>
+		<jgit-version>5.13.0.202109080827-r</jgit-version>
```

Comment:
Just a general note: Should we move those to the "master" pom instead of having two locations where we define versions / managed dependencies?

Reply:
It's fine for not widely used dependencies to be defined where they are used. E.g. jgit in extras, objectweb-asm in artifactcomparator. There is no need to define everything in single place as it makes the process of bigger changes like groupId change more expensive and there are leftovers in disconnected places.